ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/install_nvidiagpudriver.sh

```diff
@@ -2,6 +2,8 @@
 
 $COMMON_DIR/install_nvidiagpudriver.sh
 
+$UBUNTU_COMMON_DIR/install_nvidiagpudriver.sh
```

Comment:
I think you should only put 
$UBUNTU_COMMON_DIR/install_nvidiagpudriver.sh

and call,
$COMMON_DIR/install_nvidiagpudriver.sh
in first line of 
$UBUNTU_COMMON_DIR/install_nvidiagpudriver.sh